src/test/java/com/karankumar/bookproject/ui/book/BookFormTest.java

```diff
@@ -452,7 +452,7 @@ void shouldNotAllowEmptyAuthorFirstName() {
     }
 
     @Test
-    void changeAuthorNameShouldBeSaved() {
+    void shouldSaveChangedAuthorName() {
```

Comment:
As we already have 'should' in the class's display name, could you please rename this to 'saveChangedAuthorName'?